interface/wx/textctrl.h

```diff
@@ -422,6 +433,16 @@ class wxTextAttr
     */
     bool GetFontUnderlined() const;
 
+    /**
+        Returns the underline type, which is one of the @wxTextAttrUnderlineType values
+    */
+    wxTextAttrUnderlineType GetUnderlinedType() const;
+
+    /**
+        Returns the underline color used
```

Comment:
```suggestion
        Returns the underline colour used.
```
Must have period at the end of the brief description.